pkgs/tools/misc/atsi/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "atsi";
+  version = "0.0.11";
+
+  src = fetchFromGitHub {
+    owner = "queer";
+    repo = pname;
```

Comment:
`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21